### PR TITLE
bug: Map source maps to _debug directory on 'dist' compile target

### LINKED ISSUE #27: Source Maps are not being mapped correctly to '_debug' directory
This is a regression from converting to the TS2 compilation task (https://github.com/dojo/grunt-dojo2/pull/11)

### PR DESCRIPTION
**Pull request by agubler**
resolves #27

### CODE REVIEW

File: options/ts.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ function getTsTaskOptions(grunt: IGrunt, tsconfig: any): any {
 	const distCompilerOptionsOverride = tsOverrides && tsOverrides.dist && tsOverrides.dist.compilerOptions ? tsOverrides.dist.compilerOptions : {};
 	tsconfigDist.compilerOptions = Object.assign({}, tsconfig.compilerOptions, {
 		outDir: distDir,
+		mapRoot: '../dist/umd/_debug',
```
Comment (matt-gadd) [inline options/ts.ts]: should this be hardcoded?

Comment (agubler) [inline options/ts.ts]: A bigger question I support, I was effectively reverting back to the previous configuration that had regressed.

Comment (matt-gadd) [inline options/ts.ts]: shouldn't it at least use `distDir`? as the previous config would've just worked without any config right?

Comment (agubler) [inline options/ts.ts]: previously the `sourceMaps` attributes was passed exactly as has been reverted here https://github.com/dojo/grunt-dojo2/pull/11/files#diff-bd7b43d3e04f1d87f0b3d4dbf22a69cdR78

**Comment by matt-gadd** [comment]
👍

**Pull request closed without merging**